### DISCUSSION TITLE
Deploy->KnowledgePlatform->CassandraDbUpdate  failing

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#540] [Bugs]
Hi Team!
@santhosh-tg 
We are facing an issue in KnowledgePlatform CassandraDbUpdate job(release-5.1.0). below are the error logs
##################

STDERR:

/tmp/lp-data.cql:47:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:48:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:49:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:50:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:87:InvalidRequest: Error from server: code=2200 [Invalid query] message="Invalid column name outcomedeclaration because it conflicts with an existing column"

#################

We followed the below discussion, now trigger issue was resolved, but still **"Invalid column name outcomedeclaration because it conflicts with an existing column"** is there. 
https://github.com/orgs/Sunbird-Ed/discussions/143
please suggest.

Regards,
Gohila

**Comment by Krishnaj20** [comment]
@santhosh-tg @gandham-santhosh - for your attention please.

**Comment by santhosh-tg** [comment]
@gohilamariappan did you receive this error /tmp/lp-data.cql:87:InvalidRequest: Error from server: code=2200 [Invalid query] message="Invalid column name outcomedeclaration because it conflicts with an existing column" in first run of the job ? from the error it seems the table was already there @https://github.com/reshmi-nair @https://github.com/AmiableAnil FYI

**Comment by gohilamariappan** [comment]
Okay. Thank you somuch for the confirmation @santhosh-tg
